README.md

```diff
@@ -71,6 +73,14 @@ The key differences between unmanaged and managed namespaces lie in their govern
 - **list-in-ustx:** List a name for sale
 - **unlist-in-ustx:** Remove a name from sale listing
 - **buy-in-ustx:** Purchase a listed name
+## Zonefile Resolver Contract
+As mentioned above, this update to BNS-V2 introduces a new on-chain zonefile resolver contract. This is done with a single map that stores up to 4096 bytes for the zonefile for a given name & namespace. Anyone can query a name & namespace for a zonefile, but the API, Hiro's, will provide additional support for two different types of zonefile formatting: on-chain or on IPFS. 
```

Comment:
"Anyone can query a name & namespace for a zonefile, but the API, Hiro's, will provide additional support for two different types of zonefile formatting: on-chain or on IPFS." -- please remove this line. These BNSv2 docs should not make forward looking claims on behalf of 3rd parties. If / when Hiro API has support for BNSv2, you can always add in a pointer to it later. Thanks!